### ISSUE TITLE
Fehlermeldung beim Installieren

### ISSUE THREAD

**Issue by iceman-fx** [FriendsOfREDAXO/search_it#307]
Beim Versuch in einer Rex 5.10 Installation mit PHP 7.3.16 und MariaDB 5.5.62 das search_it Addon zu installieren kommt diese Meldung:

AddOn search_it konnte aus folgendem Grund nicht installiert werden:
SQL error: Error while executing statement "CREATE TABLE `rex_tmp_search_it_index` ( `id` int(11) NOT NULL auto_increment, `fid` varchar(255) , `catid` int(11) , `ftable` varchar(255) , `fcolumn` varchar(255) , `texttype` varchar(255) NOT NULL , `clang` int(11) , `filename` varchar(255) , `fileext` varchar(255) , `plaintext` longtext NOT NULL , `unchangedtext` longtext NOT NULL , `teaser` longtext NOT NULL , `values` longtext NOT NULL , `lastindexed` VARCHAR(255) NOT NULL , PRIMARY KEY (`id`), INDEX `fid` (`fid`), FULLTEXT `plaintext` (`plaintext`), FULLTEXT `unchangedtext` (`unchangedtext`), FULLTEXT `plaintext_2` (`plaintext`, `unchangedtext`) ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;"! SQLSTATE[HY000]: General error: 1214 The used table type doesn't support FULLTEXT indexes

**Comment by alxndr-w** [comment]
Deine MariaDB-Version ist allem Anschein nach zu alt. Normalerweise beginnen MariaDBs mit 10.x und MySQL mit 5.x - sicher, dass die Version stimmt?

Der Support von MySQL 5.5 wurde übrigens 2018 eingestellt - die Version ist von 2010 - ich würde mal mit dem Hoster sprechen an deiner Stelle.

Mindestanforderungen der Datenbank sollten in jedem Fall in der README genannt werden.

**Comment by tyrant88** [comment]
Ich glaube auch REX 5.10 verlangt schon eine höhere MariaDB Version...

**Comment by alxndr-w** [comment]
@tyrant88 Soweit ich weiß, wurde aber keine Prüfung eingebaut und das REDAXO lässt sich einfach so installieren.

**Comment by tyrant88** [comment]
Ja, aber search it läuft auch mit REDAXO 5.7 und da ist MySQL 5.6 oder MariaDB 10. keine Mindestvorraussetzung.

**Comment by iceman-fx** [comment]
Das ist ein System auf meinem NAS.
Aber Redaxo und praktisch alle Addons (außer search-it) ließen sich problemlos installieren und updaten.

**Comment by alxndr-w** [comment]
Da FULLTEXT-Indizes Voraussetzung sind, können wir hier nichts machen. Das NAS-System updaten wäre der Weg zum Erfolg.

**Comment by tyrant88** [comment]
Ich glaube wir haben doch indirekt die Vorraussetzung verändert, als wir die Tabelle auf InnoDB umgestellt haben...denn dafür brauch es dann MySQL 5.6...
@iceman-fx ändere mal den Tabellen-Typ in der install.sql auf MyISAM... dann müsste es sich zumindest installieren lassen.

**Comment by alxndr-w** [comment]
@iceman-fx hat das geholfen?

**Comment by alxndr-w** [comment]
PS: MySQL 5.6.10 wurde 2013-02-05 veröffentlicht. Ich finde, das darf man heutzutage selbst auf älteren Webhosting-Systemen u.ä. zwingend erwarten. 

MySQL 5.5 Support EOL war in 2018.

![image](https://user-images.githubusercontent.com/3855487/104397218-a3dd5900-554c-11eb-8f8c-9fde00a921ea.png)

**Comment by tyrant88** [comment]
Ah jetzt weiß ich, was du mit rumstänkern meintest... :-)

**Comment by iceman-fx** [comment]
Ich habe eine neuere DB installiert. Jetzt klappt es. Danke.

**Comment by alxndr-w** [comment]
Was machen wir denn nun damit? Aufwändige Prüfung vorweg, Weiche pro Datenbank-Version oder einen Hinweis in die README und die Addon-Infos zur benötigten MySQL-Version?

**Comment by iceman-fx** [comment]
Eine Prüfung und entsprechende Meldung wäre vermutlich besser.
